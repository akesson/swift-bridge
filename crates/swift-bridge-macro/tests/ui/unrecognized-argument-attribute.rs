//! # To Run
//! cargo test -p swift-bridge-macro -- ui trybuild=unrecognized-argument-attribute.rs

#[swift_bridge::bridge]
mod ffi {
    extern "Rust" {
        fn some_function(
            #[swift_bridge(InvalidArgumentAttribute)] some_value: isize
        );
    }
}

fn main() {}
