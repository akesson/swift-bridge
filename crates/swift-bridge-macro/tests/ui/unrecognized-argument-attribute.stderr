error: Unrecognized attribute "InvalidArgumentAttribute".
 --> tests/ui/unrecognized-argument-attribute.rs:8:28
  |
8 |             #[swift_bridge(InvalidArgumentAttribute)] some_value: isize
  |                            ^^^^^^^^^^^^^^^^^^^^^^^^
